test/unit/number/format.js

```diff
@@ -462,6 +462,12 @@ QUnit.test( "numbers should support significant digits in compact mode", functio
 		maximumSignificantDigits: 3,
 		minimumSignificantDigits: 1
 	})), "-127M" );
+
+	assert.equal( Globalize.formatNumber( 12849872883, {
+		minimumSignificantDigits: 1,
+		maximumSignificantDigits: 3,
+		compact: "short"
+	}), "12.8B" );
```

Comment:
Since these are unit tests, you don't have `Globalize` available. You need to use `format` (according to the above tests).

Reply:
Let's also add a test where `12850172883` becomes `12.9B`